test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/drools/FireRulesSmokeIntegrationTest.java

```diff
@@ -0,0 +1,113 @@
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.kie.cloud.integrationtests.drools;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.time.Duration;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.kie.api.KieServices;
+import org.kie.api.command.BatchExecutionCommand;
+import org.kie.api.command.Command;
+import org.kie.api.command.KieCommands;
+import org.kie.api.runtime.ExecutionResults;
+import org.kie.cloud.api.DeploymentScenarioBuilderFactory;
+import org.kie.cloud.api.scenario.WorkbenchKieServerScenario;
+import org.kie.cloud.common.provider.KieServerClientProvider;
+import org.kie.cloud.common.provider.KieServerControllerClientProvider;
+import org.kie.cloud.integrationtests.AbstractCloudIntegrationTest;
+import org.kie.cloud.integrationtests.util.WorkbenchUtils;
+import org.kie.cloud.maven.MavenDeployer;
+import org.kie.cloud.maven.constants.MavenConstants;
+import org.kie.server.api.model.KieContainerStatus;
+import org.kie.server.api.model.KieServerInfo;
+import org.kie.server.api.model.ServiceResponse;
+import org.kie.server.client.KieServicesClient;
+import org.kie.server.client.RuleServicesClient;
+import org.kie.server.controller.client.KieServerControllerClient;
+import org.kie.server.integrationtests.shared.KieServerAssert;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class FireRulesSmokeIntegrationTest extends AbstractCloudIntegrationTest<WorkbenchKieServerScenario> {
+
+    private static final Logger logger = LoggerFactory.getLogger(DroolsSessionFailoverIntegrationTest.class);
+
+    private KieServerControllerClient kieServerControllerClient;
+    private KieServicesClient kieServerClient;
+    private RuleServicesClient kieServerRuleServiceClient;
+
+    private static KieCommands commandsFactory = KieServices.Factory.get().getCommands();
+
+    private static final String LIST_NAME = "list";
+    private static final String LIST_OUTPUT_NAME = "output-list";
+    private static final String KIE_SESSION = "defaultKieSession";
+    private static final String FAST_RULE = "Fast rule executed";
+    private static final String SLOW_RULE = "Slow rule executed";
+
+    @Override
+    protected WorkbenchKieServerScenario createDeploymentScenario(DeploymentScenarioBuilderFactory deploymentScenarioFactory) {
+        return deploymentScenarioFactory.getWorkbenchKieServerScenarioBuilder()
+                .withExternalMavenRepo(MavenConstants.getMavenRepoUrl(), MavenConstants.getMavenRepoUser(), MavenConstants.getMavenRepoPassword())
+                .build();
+    }
+
+    @Before
+    public void setUp() {
+        MavenDeployer.buildAndDeployMavenProject(ClassLoader.class.getResource("/kjars-sources/rule-project").getFile());
```

Comment:
Rule in this project is quite time consuming, could you consider using some other rules to speed up test?